enroll/service.go

```diff
@@ -0,0 +1,100 @@
+package enroll
+
+import "io/ioutil"
+
+type Service interface {
+	Enroll() (Profile, error)
+}
+
+func NewService(pushCertPath string, pushCertPass string, caCertPath string) (Service, error) {
+	pushTopic, err := GetPushTopicFromPKCS12(pushCertPath, pushCertPass)
+	if err != nil {
+		return nil, err
+	}
+
+	var caCert []byte
+
+	if caCertPath != "" {
+		caCert, err = ioutil.ReadFile(caCertPath)
+
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	scepSubject := [][][]string{
+		[][]string{
+			[]string{"O", "MicroMDM"},
+			[]string{"CN", "MDM Identity Certificate:UDID"},
+		},
+	}
+
+	return &service{
+		Url:         "https://micromdm.local:6443",
```

Comment:
The URL for the service and SCEP server should be configurable using a flag in main.